app/models/user.rb

```diff
@@ -8,7 +8,7 @@ class User < CouchRest::Model::Base
   property :password_salt, String, :accessible => true
   property :contact_email, String, :accessible => true
   property :contact_email_key, String, :accessible => true
-
+  property :invite_code, String, :accessible => true
```

Comment:
This probably means we store the invite code with the user, right?

I think we should only keep it during registration and then forget about it. But I am not sure. @elijh, what do you think? Do we want to know which user used which invite code?